lib/src/test/java/io/ably/lib/test/realtime/RealtimeInitTest.java

```diff
@@ -94,7 +94,6 @@ public void init_host() {
 			ClientOptions opts = new ClientOptions(testVars.keys[0].keyStr);
 			opts.restHost = "some.other.host";
 			ably = new AblyRealtime(opts);
-			assertEquals("Unexpected host mismatch", Defaults.getHost(opts), opts.restHost);
```

Comment:
So this worries me on two counts.  Firstly, if you ave a test without an assertion, this is not a test.  Secondly, this is a realtime test which is supposed to test that when the REST host is set, all rest requests go to that host.  Equally, if a realtimeHost is set, then all realtime requests go to that host.  We need that test as I am not confident that is working without suitable test coverage.